packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -568,3 +569,143 @@ class _TextSelectionHandleOverlayState extends State<_TextSelectionHandleOverlay
     return null;
   }
 }
+
+/// A gesture detector to detect concurrent gesture events for a text field.
```

Comment:
I'm not sure that "concurrent" is the best way to describe the events this widget recognizes.  The next paragraph calls them event chains, maybe: A gesture detector that recognizes all of the individual tap up/down and double tap down events in a chain of events.

Reply:
Reworded